### PR TITLE
Added Batch Request

### PR DESCRIPTION
**Pull request by AliabbasMerchant**
@YourTechBud  Done

### CODE REVIEW

**Comment by YourTechBud** [comment]
@AliabbasMerchant you think we should throw an exception for batch.add? Maybe just returb a false if the batch add failed? Im not sure. What do you think?

Also, did you test if it works?

**Comment by AliabbasMerchant** [comment]
Maybe returning false would be a good idea, but then you would have to exclusively check for the false value every time. An exception would just halt the execution (unless caught), and you can directly write all your multiple statements in a try catch-block.

I haven't tested the batch thing, but assuming that the rest of CRUD works and knowing that the server works, I am hoping that this also works 🤞

File: src/main/java/com/spaceuptech/space_api/mongo/Delete.java [hunk 1]
```diff
@@ -1,21 +1,23 @@
...
+import com.spaceuptech.space_api.proto.Meta;
+import com.spaceuptech.space_api.utils.*;
 
 import java.util.HashMap;
 
 public class Delete {
 
     private Meta meta;
-    private String operation;
+    private String operation, db, collection;
```
Comment (YourTechBud) [inline src/main/java/com/spaceuptech/space_api/mongo/Delete.java]: No need to store db and collection. Its already there in the meta object

**Review by YourTechBud — CHANGES_REQUESTED** [review]
Minor changes. It was an amazing idea to use function overloading!!

File: src/main/java/com/spaceuptech/space_api/mongo/Delete.java (hunk 1 above)
Comment (AliabbasMerchant) [reply src/main/java/com/spaceuptech/space_api/mongo/Delete.java]: Right. Cool

**Comment by AliabbasMerchant** [comment]
@YourTechBud Done

**Pull request merged**